config/src/main/resources/holesky.json

```diff
@@ -827,196 +826,196 @@
+      "balance": "0x52b7d2dcc80cd2e4000000"
+    },
+    "0x9E415A096fF77650dc925dEA546585B4adB322B6": {
+      "balance": "0xd3c21bcecceda1000000"
+    },
+    "0xA0766B65A4f7B1da79a1AF79aC695456eFa28644": {
+      "balance": "0xd3c21bcecceda1000000"
+    },
+    "0xA29B144A449E414A472c60C7AAf1aaFfE329021D": {
+      "balance": "0xd3c21bcecceda1000000"
+    },
+    "0xa55395566b0b54395B3246f96A0bDc4b8a483df9": {
+      "balance": "0xd3c21bcecceda1000000"
+    },
+    "0xAC9ba72fb61aA7c31A95df0A8b6ebA6f41EF875e": {
+      "balance": "0xd3c21bcecceda1000000"
+    },
+    "0xB0498C15879db2eE5471d4926c5fAA25C9a09683": {
+      "balance": "0xd3c21bcecceda1000000"
+    },
+    "0xB19Fb4c1f280327e60Ed37b1Dc6EE77533539314": {
+      "balance": "0x52b7d2dcc80cd2e4000000"
+    },
+    "0xC21cB9C99C316d1863142F7dD86dd5496D81A8D6": {
+      "balance": "0xd3c21bcecceda1000000"
+    },
+    "0xc473d412dc52e349862209924c8981b2ee420768": {
+      "balance": "0xd3c21bcecceda1000000"
+    },
+    "0xC48E23C5F6e1eA0BaEf6530734edC3968f79Af2e": {
+      "balance": "0x52b7d2dcc80cd2e4000000"
+    },
+    "0xc6e2459991BfE27cca6d86722F35da23A1E4Cb97": {
+      "balance": "0x52b7d2dcc80cd2e4000000"
+    },
+    "0xD3994e4d3202dD23c8497d7F75bF1647d1DA1bb1": {
+      "balance": "0x19D971E4FE8401E74000000"
+    },
+    "0xDCA6e9B48Ea86AeBFDf9929949124042296b6e34": {
+      "balance": "0xd3c21bcecceda1000000"
+    },
+    "0xe0a2Bd4258D2768837BAa26A28fE71Dc079f84c7": {
+      "balance": "0x52b7d2dcc80cd2e4000000"
+    },
+    "0xEA28d002042fd9898D0Db016be9758eeAFE35C1E": {
+      "balance": "0xd3c21bcecceda1000000"
+    },
+    "0xEfA7454f1116807975A4750B46695E967850de5D": {
+      "balance": "0xd3c21bcecceda1000000"
+    },
+    "0xFBFd6Fa9F73Ac6A058E01259034C28001BEf8247": {
+      "balance": "0x52b7d2dcc80cd2e4000000"
+    },
+    "0xe0991E844041bE6F11B99da5b114b6bCf84EBd57": {
+      "balance": "0xd3c21bcecceda1000000"
+    },
+    "0x15E719b6AcAf1E4411Bf0f9576CB1D0dB161DdFc": {
+      "balance": "0xd3c21bcecceda1000000"
+    },
+    "0x346D827a75F98F0A7a324Ff80b7C3F90252E8baC": {
+      "balance": "0xd3c21bcecceda1000000"
+    },
+    "0x73b2e0E54510239E22cC936F0b4a6dE1acf0AbdE": {
+      "balance": "0x52b7d2dcc80cd2e4000000"
+    },
+    "0xBb977B2EE8a111D788B3477D242078d0B837E72b": {
+      "balance": "0xd3c21bcecceda1000000"
+    },
+    "0x834Dbf5A03e29c25bc55459cCe9c021EeBE676Ad": {
+      "balance": "0xd3c21bcecceda1000000"
+    },
+    "0xD1F77E4C1C45186e8653C489F90e008a73597296": {
+      "balance": "0xd3c21bcecceda1000000"
+    },
+    "0xb04aeF2a3d2D86B01006cCD4339A2e943d9c6480": {
+      "balance": "0xd3c21bcecceda1000000"
+    },
+    "0xC9CA2bA9A27De1Db589d8c33Ab8EDFa2111b31fb": {
+      "balance": "0xd3c21bcecceda1000000"
+    },
+    "0x4BC656B34De23896fa6069C9862F355b740401aF": {
+      "balance": "0x084595161401484a000000"
     }
   },
   "coinbase": "0x0000000000000000000000000000000000000000",
   "baseFeePerGas": "0x3B9ACA00",
```

Comment:
Why does the Besu config need `baseFeePerGas` but the default genesis.json doesn't?

Reply:
Just double checked and we technically don't because it is the default initial value: https://github.com/hyperledger/besu/blob/main/config/src/main/java/org/hyperledger/besu/config/GenesisConfigFile.java#L43

However, if we change it in this PR then we will no longer exactly match https://github.com/eth-clients/holesky/blob/main/custom_config_data/besu.json if anyone cares about that.

I actually prefer it being in, more explicit, since it is actually part of the genesis block.